### PR TITLE
Allow users to configure autopause settings for cluster

### PR DESCRIPTION
**Pull request by akarukappadath**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -694,6 +698,12 @@ definitions:
           Defaults to false.
         type: boolean
         default: false
+      autopause:
+        type: boolean
+        description: Whether autopause feature is enabled for this specific cluster.
```
Comment (rtitle) [inline src/main/resources/swagger/api-docs.yaml]: Can you also say: If unset, autopause will be enabled and a system default threshold will be used.

Comment (kyuksel) [reply src/main/resources/swagger/api-docs.yaml]: should that default be mentioned somewhere, too?

Comment (rtitle) [reply src/main/resources/swagger/api-docs.yaml]: We could say "a system default of 30 minutes". If we change the default however, we need to remember to change the Swagger doc. OTOH, if the user cares about the value they should probably specify the value themselves. :)

Comment (kyuksel) [reply src/main/resources/swagger/api-docs.yaml]: fair enough

**Pull request merged**